readme.md

```diff
@@ -153,12 +153,14 @@ The primary goals of this textbook are to solidify the grammar, vocabulary and k
 * [Japanese in Anime & Manga](http://anime-manga.jp/) - Learn Japanese from anime and manga. 
 * [Satori Reader](https://www.satorireader.com/) - Japanese reading and listening practice, first two chapters of each article is free.:moneybag: 
 * [Keio University Reading Materials Bank](http://language.tiu.ac.jp/materials/jpn/index.html) - The site is mostly in Japanese, but follow the 1-stars for the easiest reading materials. It's great because there is an English glossary integrated into the articles.
+* [Jim Breen's academic website at Monash](http://nihongo.monash.edu/japanese.html) has some links to literature.
```

Comment:
Probably it's better to add the literature link directly instead of link to the wall of text on his website. You can credit him for the link in description